src/Teams.Web/Controllers/ManageSprintsController.cs

```diff
@@ -142,34 +132,44 @@ public async Task<IActionResult> EditSprintAsync(int teamId, int sprintId, strin
             }
-            if(activeSprints != null && isActive == true)
+
+            if (currentSprint.Name == sprintName && currentSprint.DaysInSprint == daysInSprint && currentSprint.StoryPointInHours == storePointsInHours && currentSprint.Status == status)
             {
-                return RedirectToAction("EditSprint", new { teamId = teamId, sprintId = sprintId, errorMessage = _localizer["ActiveFieldError"] });
+                return RedirectToAction("EditSprint", new { teamId = teamId, sprintId = sprintId, errorMessage = _localizer["HasntAnyChange"] });
             }
 
-            var sprint = new Sprint { Id = sprintId, TeamId = teamId, Name = sprintName, DaysInSprint = daysInSprint, StoryPointInHours = storePointsInHours, IsActive = isActive };
-            var result = await EditSprintAsync(sprint);
-
-            if (result)
+            if ((currentSprint.Status == 2 && status == 1) || (currentSprint.Status == 1 && status == 0) || currentSprint.Status == status)
             {
-                return RedirectToAction("AllSprints", new { teamId = teamId });
+                var sprint = new Sprint { Id = sprintId, TeamId = teamId, Name = sprintName, DaysInSprint = daysInSprint, StoryPointInHours = storePointsInHours, Status = status };
+                var result = await EditSprintAsync(sprint);
+
+                if (result)
+                {
+                    return RedirectToAction("AllSprints", new { teamId = teamId });
+                }
+                else
+                {
+                    return RedirectToAction("AddError", new { teamId = teamId });
+                }
             }
             else
             {
-                return RedirectToAction("AddError", new { teamId = teamId });
+                return RedirectToAction("EditSprint", new { teamId = teamId, sprintId = sprintId, errorMessage = _localizer["CantChangeStatus"] });
             }
-
         }
 
         [HttpPost]
         [Authorize]
-        public async Task<IActionResult> AddSprintAsync(int teamId, string sprintName, int daysInSprint, int storePointsInHours, bool isActive)
+        public async Task<IActionResult> AddSprintAsync(int teamId, string sprintName, int daysInSprint, int storePointsInHours, int status)
         {
-            var sprint = new Sprint { TeamId = teamId, Name = sprintName, DaysInSprint = daysInSprint, StoryPointInHours = storePointsInHours, IsActive = isActive };
-            
+            var sprint = new Sprint { TeamId = teamId, Name = sprintName, DaysInSprint = daysInSprint, StoryPointInHours = storePointsInHours, Status = status };
+
             var Sprints = await _manageSprintsService.GetAllSprintsAsync(teamId, new DisplayOptions());
-            var activeSprint = Sprints.FirstOrDefault(i => i.IsActive == true);
+            var activeSprint = Sprints.FirstOrDefault(i => i.Status == 1);
```

Comment:
Maybe we need to store Status values as a constants? It is not clear from the code what 1 mean

Reply:
i have fixed this. Please look at the new commit